package.json

```diff
@@ -33,7 +33,7 @@
   "scripts": {
     "test": "npm run test-server && npm run test-js",
     "test-server": "make test-server",
-    "test-js": "run-with-testrpc --port 8544 --gas-limit 8000000 --defaultBalanceEther 1000 'npx truffle --network npmtest test'",
+    "test-js": "run-with-testrpc --port 8544 --gas-limit 8000000 --defaultBalanceEther 1000 --deterministic 'npx truffle --network npmtest test'",
```

Comment:
why did you add deterministic? we preferred not to rely on specific addresses..

Reply:
Good catch - this is from a previous version of the test suite where I included the private key for one of the relays, but the current tests don't need the key to correspond to an actual funded address.

Reply:
Some of the penalization tests require having access to a relay's private key in order to sign transactions manually (since ganache's RPC interface doesn't support doing so).

I added back a test in e22dbba80841a6f60dc24b51a86fe72f76eeb1b2 checking that this key matches the address. I don't see much of an issue in using deterministic mode in these tests to be honest, in fact, I'd rather the test run to be reproducible as opposed to based on random values.